### PR TITLE
Add guides for installing GHC and Cabal

### PR DESCRIPTION
**Pull request by jacobstanley**
I think it makes sense to have these guides in the mafia repo so non-Ambiata users can see how we expect the system to be configured.

/cc @markhibberd @nhibberd @charleso @tranma @thumphries @HuwCampbell

### CODE REVIEW

**Comment by markhibberd** [comment]
👍

**Comment by tranma** [comment]
this is good 🍨

**Comment by erikd-ambiata** [comment]
:fish:

**Comment by HuwCampbell** [comment]
Thanks @jystic

File: README.md [hunk 1]
```diff
@@ -372,3 +375,14 @@ The same as `cabal update`, but limited to retrieving a new index
...
+
+System configuration
+--------------------
+
+Mafia expects both GHC and Cabal to be installed and on the `PATH`.
+
+Follow the guides below to configure your system correctly:
+
+- [Installing GHC](https://github.com/ambiata/mafia/blob/master/doc/ghc.md)
+- [Installing Cabal](https://github.com/ambiata/mafia/blob/master/doc/cabal.md)
```
Comment (HuwCampbell) [inline README.md:388]: Maybe add the switch link here too?

Comment (jacobstanley) [reply README.md:388]: It's an anchor inside the ghc doc, so you would find it by clicking on this link, that's close enough for me

**Pull request merged**